tiledb/sm/misc/logger.cc

```diff
@@ -33,6 +33,7 @@
 #include "tiledb/sm/misc/logger.h"
 
 #include <spdlog/sinks/stdout_sinks.h>
+#include <spdlog/sinks/stdout_color_sinks.h>
```

Comment:
Please make this conditional on `SPDLOG_VERSION >= 10500`

Reply:
(see calculation here: https://github.com/gabime/spdlog/blob/616caa5d30172b65cc3a06800894c575d70cb8e6/include/spdlog/version.h)

Reply:
Hmm, actually it looks like this was added in 1.0, so it can be `SPDLOG_VERSION >= 10000` instead.